### DISCUSSION TITLE
Joining a voice channel by numeric code

### DISCUSSION THREAD

**Discussion by whyrmph** [Rapptz/discord.py#8033] [Q&A]
Hello, I have a question, please tell me how to implement slash command as in the screenshot.
(I mean **/join number**).
I will be extremely grateful.
Thanks in advance for your reply.

p.s i translated the screenshot in google translator for better understanding
![Screenshot_1 translated](https://user-images.githubusercontent.com/57942975/168135056-48f03dce-fff4-46fe-b9bf-e33c7964a9d6.jpg)
.

**Comment by RGBCube** [comment]
You will need to assign the number a channel ID in the bot, and then get the channel ID that number is assigned to when the command is called with the number. Then you simply connect to the channel using the ID
